.gitattributes

```diff
@@ -1,2 +1,3 @@
 *.md text=auto eol=lf
-*.sh text=auto eol=lf
\ No newline at end of file
+*.sh text=auto eol=lf
+build.zig text=auto eol=lf
```

Comment:
out of scope for this PR (because it might have unknown unknowns), but it would be great to simplify this to just

````
* text=auto eol=lf
```